### ISSUE TITLE
Missing OpenRAM copyright/license

### ISSUE THREAD

**Issue by mguthaus** [lekez2005/opennvram-reram#9]
Hi,

A large portion of this code is based on openram and distributed under the BSD license. The requirements of the BSD license require that you distribute with that license. However, it seems that you removed the copyright. Please add it back.

**"1. Redistributions of source code must retain the above copyright notice, this
   list of conditions and the following disclaimer."**
   
Thanks,
Matt

**Comment by mguthaus** [comment]
This would also include copyright on individual files that seem to have been stripped.

**Comment by lekez2005** [comment]
Hi Prof. Guthaus,

The current license is at https://github.com/lekez2005/opennvram-reram/blob/main/LICENSE which is still under BSD and includes the original copyrights. I added the Cornell University copyright at the top, is that inappropriate?

With respect to the copyright on individual files, I branched off since 2018, when the individual files didn't have copyright headers. As I previously said in #1, this repo was explicitly created for the Skywater tape-out. Now that you mention it, I will be publishing the full repo which will include the entire git history within the next week. Would it be okay with you to delay the discussion about individual files until then?
